frontend/src/utils/utils.ts

```diff
@@ -295,3 +296,28 @@ export function emptyStringToNull(value: any, originalValue: any) {
 export const clearanceNotificationSentOnRequired = (statusCode: string) => {
   return ['ERP-ON', 'ERP-OH'].includes(statusCode);
 };
+
+/**
+ * Determine if the specified error is an axios error, improves intellisense.
+ * @param error The error being checked.
+ */
+export const isAxiosError = (err: any): err is AxiosError => {
+  if (!err) return false;
+  return (err as AxiosError).isAxiosError !== undefined;
+};
+
+/** generate administrative area ready for use in api endpoints */
+export const toApiAdminArea = (adminArea: IAdministrativeArea, name?: string): IApiAdminArea => {
```

Comment:
Doesn't look like a global util, more like a specific use-case for the Admin components.  I've move to a util file in the relevant area.